### PR TITLE
Add basic copr options

### PR DESCRIPTION
**Pull request by FrNecas**
Resolves: #708 
Related issue: #461 (adds a mechanism for passing options to builders)

### CODE REVIEW

File: rebasehelper/plugins/build_tools/rpm/copr_.py [hunk 1]
```diff
@@ -41,6 +41,21 @@ class Copr(BuildToolBase):
     Class representing Copr build tool.
     """
 
+    OPTIONS = [
+        {
+            'name': ['--remove-copr-project-after'],
+            'default': 60,
+            'help': 'after how many days the created copr build will be '
+                    'deleted, -1 for never, defaults to %(default)s',
```
Comment (nforro) [inline rebasehelper/plugins/build_tools/rpm/copr_.py]: I think a simple switch would be sufficient, something like `--copr-project-permanent`, defaulting to `False`.

File: rebasehelper/plugins/build_tools/rpm/copr_.py [hunk 2]
```diff
@@ -41,6 +41,21 @@ class Copr(BuildToolBase):
...
+        },
+        {
+            'name': ['--visible-copr-project'],
```
Comment (nforro) [inline rebasehelper/plugins/build_tools/rpm/copr_.py]: `--copr-project-frontpage` sounds a bit better to me.

File: rebasehelper/plugins/build_tools/rpm/copr_.py (hunk 1 above)
Comment (FrNecas) [reply rebasehelper/plugins/build_tools/rpm/copr_.py]: You are right that this is too complex. The number of days can be later changed in the web interface. So it should either be permanent or last 60 days?

Comment (nforro) [reply rebasehelper/plugins/build_tools/rpm/copr_.py]: Exactly.

File: rebasehelper/helpers/copr_helper.py [hunk 3]
```diff
@@ -52,11 +52,13 @@ def get_client(cls):
             return client
 
     @classmethod
-    def create_project(cls, client, project, chroot, description, instructions):
+    def create_project(cls, client, project, chroot, description, instructions, permanent=False, hide=True):
         try:
             client.project_proxy.add(ownername=client.config.get('username'),
                                      projectname=project,
                                      chroots=[chroot],
+                                     delete_after_days=None if permanent else 60,
```
Comment (nforro) [inline rebasehelper/helpers/copr_helper.py]: I would rather see a constant.

File: rebasehelper/plugins/build_tools/rpm/copr_.py [hunk 4]
```diff
@@ -41,6 +41,21 @@ class Copr(BuildToolBase):
     Class representing Copr build tool.
     """
 
+    OPTIONS = [
+        {
+            'name': ['--copr-permanent-project'],
```
Comment (nforro) [inline rebasehelper/plugins/build_tools/rpm/copr_.py]: Why not `--copr-project-permanent`?

Comment (FrNecas) [reply rebasehelper/plugins/build_tools/rpm/copr_.py]: I meant permanent as an adjective tied to project. Though you are right, that `--copr-project-permanent` is better, it implies that bot of the added options influence the project.

File: rebasehelper/plugins/build_tools/rpm/copr_.py [hunk 5]
```diff
@@ -41,6 +41,21 @@ class Copr(BuildToolBase):
     Class representing Copr build tool.
     """
 
+    OPTIONS = [
```
[the remark this answers is not in the document]
Comment (FrNecas) [reply rebasehelper/plugins/build_tools/rpm/copr_.py]: I will fix it in the PR where I will enable specifying chroots

**Pull request merged**